### PR TITLE
US48, US49: Create auction bidding rule views

### PR DESCRIPTION
**Pull request by tomejzen**

### CODE REVIEW

File: Web/Views/AuctionBiddingRules/Edit.cshtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+            <div class="form-group">
+                <input type="submit" value="Zapisz" class="btn btn-default" />
+            </div>
+        </form>
+    </div>
+</div>
+
+<div>
+    <a href='javascript:history.go(-1)'>Wróć</a>
```
Comment (SzymonPuszcz) [inline Web/Views/AuctionBiddingRules/Edit.cshtml]: What if user opens page in new tab?

Comment (tomejzen) [reply Web/Views/AuctionBiddingRules/Edit.cshtml]: da ting goes skraa

Comment (krzysiekprzekwas) [reply Web/Views/AuctionBiddingRules/Edit.cshtml]: XD

Comment (tomejzen) [reply Web/Views/AuctionBiddingRules/Edit.cshtml]: Removed.

**Pull request merged**